### DISCUSSION TITLE
Incoming Email Issue

### DISCUSSION THREAD

**Discussion by arunkuttytry** [postalserver/postal#2851] [Help with using Postal]
Hi all,

i have facing issue incoming email routing, ill setup  routing opt but user got bounce , i have check smtp  check mxtool, i got the error how we fix this 
![image](https://github.com/postalserver/postal/assets/107031529/b825e065-9ef1-4a4b-9445-387e2e15037d)

**Comment by willpower232** [comment]
If it is unable to connect, perhaps your DNS is wrong, you have a firewall, or Postal has failed to start or isn't currently running.

**Reply by arunkuttytry** [reply]
how to im fix this

**Reply by willpower232** [reply]
if you can see with `postal status` that everything is working then you can check the DNS is correct with https://whatsmydns.net and if it is, then you can look for firewalls which are blocking connections to your Postal.

**Comment by LotisHunters** [comment]
What version of Postal are you on? There were some configuration issues for incoming email that have been fixed with 3.1.0
